app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -474,57 +468,36 @@ private fun ConversationScreenContent(
         membersToMention = membersToMention
     )
 
-    SnackBarMessage(snackbarMessage, conversationState, conversationScreenState, onSnackbarMessageShown)
+    SnackBarMessage(composerMessages, conversationMessages, conversationScreenState)
```

Comment:
I think this `SnackBarMessage` could be used in `ConversationScreen` where `conversationScreenState` is created and we wouldn't have to pass both Flows all the way to the `ConversationScreenContent`

Reply:
Thanks and done.